translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\tthat exist off platform, on other clusters, or locally."
+msgstr ""
+"\n"
+"\tCria um serviço com ExternalName com o nome especificado.\n"
+"\n"
+"\tServiço ExternalName referencia um endereço externo de DNS ao invés de\n"
+"\tapenas pods, o que permite aos desenvolvedores de aplicações referenciar "
+"serviços\n"
+"\tque existem fora da plataforma, em outros clusters ou localmente."
+
+#: pkg/kubectl/cmd/help.go:30
+msgid ""
+"\n"
+"\tHelp provides help for any command in the application.\n"
+"\tSimply type kubectl help [path to command] for full details."
+msgstr ""
+"\n"
+"\tHelp provê ajuda para qualquer comando no aplicação.\n"
+"\tDigite simplesmente kubectl help [caminho do comando] para detalhes "
+"completos."
+
+#: pkg/kubectl/cmd/create_service.go:173
+msgid ""
+"\n"
+"    # Create a new LoadBalancer service named my-lbs\n"
+"    kubectl create service loadbalancer my-lbs --tcp=5678:8080"
+msgstr ""
+"\n"
+"    # Cria um novo serviço do tipo LoadBalancer chamado my-lbs\n"
+"    kubectl create service loadbalancer my-lbs —tcp=5678:8080"
+
+#: pkg/kubectl/cmd/create_service.go:53
+msgid ""
+"\n"
+"    # Create a new clusterIP service named my-cs\n"
+"    kubectl create service clusterip my-cs --tcp=5678:8080\n"
+"\n"
+"    # Create a new clusterIP service named my-cs (in headless mode)\n"
+"    kubectl create service clusterip my-cs --clusterip=\"None\""
+msgstr ""
+"\n"
+"    # Cria um novo serviço clusterIP chamado my-cs\n"
+"    kubectl create service clusterip my-cs --tcp=5678:8080\n"
+"\n"
+"    # Cria um novo serviço clusterIP chamado my-cs (em modo headless)\n"
+"    kubectl create service clusterip my-cs --clusterip=\"None\""
+
+#: pkg/kubectl/cmd/create_deployment.go:36
+msgid ""
+"\n"
+"    # Create a new deployment named my-dep that runs the busybox image.\n"
+"    kubectl create deployment my-dep --image=busybox"
+msgstr ""
+"\n"
+"    # Cria um novo deployment chamado my-dep que executa uma imagem "
+"busybox.\n"
+"    kubectl create deployment my-dep —image=busybox"
+
+#: pkg/kubectl/cmd/create_service.go:116
+msgid ""
+"\n"
+"    # Create a new nodeport service named my-ns\n"
+"    kubectl create service nodeport my-ns --tcp=5678:8080"
+msgstr ""
+"\n"
+"    # Cria um novo serviço nodeport chamado my-ns\n"
+"    kubectl create service nodeport my-ns —tcp=5678:8080"
+
+#: pkg/kubectl/cmd/clusterinfo_dump.go:62
+msgid ""
+"\n"
+"    # Dump current cluster state to stdout\n"
+"    kubectl cluster-info dump\n"
+"\n"
+"    # Dump current cluster state to /path/to/cluster-state\n"
+"    kubectl cluster-info dump --output-directory=/path/to/cluster-state\n"
+"\n"
+"    # Dump all namespaces to stdout\n"
+"    kubectl cluster-info dump --all-namespaces\n"
+"\n"
+"    # Dump a set of namespaces to /path/to/cluster-state\n"
+"    kubectl cluster-info dump --namespaces default,kube-system --output-"
+"directory=/path/to/cluster-state"
+msgstr ""
+"\n"
+"    # Despeja o estado corrente do custer para o stdout\n"
```

Comment:
hum, 'despejar' in portuguese for me have the meaning of 'garbage things'.

Maybe instead of 'despeja' we can change it to 'Exibe', but honestly no strong opinion in this

Reply:
Hum, saw in some translation below that you used the dump word with "Coleta" instead of "despeja" and I've liked. Something like "Coleta o estado corrente do cluster e exibe no stdout" seems really better :D